### ISSUE TITLE
Freemius Performance issue after upgrading to 2.1.5

### ISSUE THREAD

**Issue by mattburesh** [TablePress/TablePress#178]
We use TablePress on a large-ish WordPress multisite installation, and after upgrading to 2.1.5 this morning we noticed a performance issue that looks like it's related to TablePress' Freemius dependency.

What I was able to gather from New Relic was that Freemius is trying to schedule a cron job, and in the process it tries to create a map of all of the sites in our network. It never seemed to stop doing this, so it ate up a bunch of resources.

I'm not sure of the root cause, but we were able to roll back to 2.1.3 and that "solved" the issue. I'll be opening a ticket with Freemius as well, but wanted to make you aware of the issue too.

**Comment by TobiasBg** [comment]
Hi @mattburesh,

thanks for opening this issue! I haven't received similar reports yet (well, maybe that's because not too many users have a large multisite), so that's highly appreciated!

You mention that you reverted back to 2.1.3. Is that simply because that's the version where you had updated from or is there any reason that you left out 2.1.4 here?
TablePress 2.1.3 is using Freemius SDK 2.5.7, TablePress 2.1.4 uses 2.5.8, and TablePress 2.1.5 is up-to-date with 2.5.10.

And yes, ultimately, this is something that will need to be looked at and fixed in the SDK by Freemius, so it would be great if you could already notify the Freemius team about this issue (they might prefer via email to support@freemius.com -- feel free to put me in CC, of course).

Best wishes,
Tobias

**Comment by mattburesh** [comment]
That's correct, we wanted to roll back to the version we were successfully running before, just to help narrow down the cause.

I'm talking with someone from Freemius now, I can keep you posted if they end up finding anything.

**Comment by TobiasBg** [comment]
Hi @mattburesh,

great! Good to hear that Freemius is looking into this! Yes, it would be great if you could post an update here then!
I'll of course update to a new version of the SDK asap, once they maybe release a fix!

In the mean time, I'll close this issue, as there's nothing really actionable to do for me here, at the moment. If you feel otherwise, please don't hesitate to reopen!

Best wishes,
Tobias
